examples/bridge-app/esp32/main/main.cpp

```diff
@@ -323,10 +323,10 @@ void HandleDeviceStatusChanged(Device * dev, Device::Changed_t itemChangedMask)
 
     if (itemChangedMask & Device::kChanged_Name)
     {
-        uint8_t zclName[kUserLabelSize];
-        ToZclCharString(zclName, dev->GetName(), kUserLabelSize - 1);
+        uint8_t zclName[kNodeLabelSize];
+        ToZclCharString(zclName, dev->GetName(), kNodeLabelSize - 1);
```

Comment:
`kNodeLabelSize` is defined as 32.

However, here, the buffer is 32, but has to lose 1 byte for the length prefix. That means the max label length is 31, which is out of spec.

There may be a corner case here of how storage is allocated in attribute store, versus the lengths stated in Matter spec, which do not account for overhead, just for actual data length).

Reply:
Change the length of the temporary buffer to kNodeLabelSize+1.